includes/class-wcpdf-admin.php

```diff
@@ -236,6 +237,10 @@ public function add_listing_actions( $order ) {
 	 * @param array $columns shop order columns
 	 */
 	public function add_invoice_columns( $columns ) {
+		if ( OrderUtil::custom_orders_table_usage_is_enabled() && isset( $_REQUEST['page'] ) && $_REQUEST['page'] == 'wc-orders' && isset( $_REQUEST['action'] ) && $_REQUEST['action'] == 'edit' ) {
```

Comment:
Will also need the compatibility class for users that run an older WooCommerce version.